src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,9 @@
 
 // Third party includes
 
+// Marks handler parameter as unused and prevents compiler warnings.
+#define UNUSED_ARG(arg) (void)(arg)
```

Comment:
I don't know why... but i don't like unnecessary defines in our code :) I prefer not to use it where there is no actual need